src/app/components/Spinner/index.module.scss

```diff
@@ -0,0 +1,15 @@
+.spinner {
+    @apply tw-flex tw-items-center tw-justify-center tw-text-gray-7;
+    svg {
+        min-width: 10px;
+        min-height: 10px;
+        max-width: 100px;
+        max-height: 100px;
```

Comment:
```suggestion
        min-width: 0.625rem;
        min-height: 0.625rem;
        max-width: 6.25rem;
        max-height: 6.25rem;
```

Reply:
changed, we will use "rem" everywhere instead of "px"?

Reply:
Almost everywhere :D Or better put - everywhere where it makes sense (and it does make sense for almost everything but you don't want for instance border width to be wider on bigger screens so it's completely ok to use `px` there).